package.json

```diff
@@ -62,5 +64,8 @@
   "browserslist": [
     "extends browserslist-config-kolibri"
   ],
-  "dependencies": {}
+  "dependencies": {},
+  "volta": {
+    "node": "16.18.0"
+  }
```

Comment:
Been meaning to ask if we can just do this since we suggest volta in our dev docs (and likewise pin node@10 in the release 15 branch) @rtibbles

Reply:
Yes, this seems eminently sensible to me.